requirements.txt

```diff
@@ -2,6 +2,7 @@ altair>=4.0.0,<5
 black==19.10b0
 Click>=7.0
 future>=0.16
+google-cloud-storage>=1.28.0
```

Comment:
I don't think we want to add this to requirements.

@jcampbell , how are we handling optional requirements these days?

Reply:
Three ways:

1. Part of a major feature set/backend (e.g. sqlalchemy or pyspark). In that case, it deserves a section in extras_require. I like the idea of adding a 'gcp' section to extras_require.
2. We *do* add them to requirements-dev.txt to ensure they're available for required tests. (not relevant here because already done).
3. We wrap optional dependencies in try/catch with a warning that it is missing in modules that use it (e.g. a store backend). (not relevant here since this has already been handled)